perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -903,6 +903,28 @@ sub report_edit : Path('report_edit') : Args(1) {
 
         if ( $problem->state ne $old_state ) {
             $c->forward( 'log_edit', [ $id, 'problem', 'state_change' ] );
+
+            unless (
+              $problem->state eq 'unconfirmed' || $old_state eq 'unconfirmed' ||
+              $problem->state eq 'hidden' || $old_state eq 'hidden'
+            ) {
+                my $name = _('admin');
```

Comment:
I think I'd go for "an administrator" here (and also change the untranslated similar string in Moderate.pm to match this).